packages/react/src/components/Slider/Slider.tsx

```diff
@@ -624,6 +779,7 @@ export default class Slider extends PureComponent {
                 <span className={`${prefix}--slider__range-label`}>
                   {formatLabel(min, minLabel)}
                 </span>
+                {/* @ts-ignore onBlur + onChange types are incompatible*/}
```

Comment:
is it correct that the `onBlur` and `onChange` functions are passed onto the top `div` with `{...other}` in line 794 @tay1orjones? seems like  we're calling those prop functions explicitly in our`onBlur` and  `componentDidUpdate` functions and the types do seem to be incompatible 🤔. should these props be deleted from `other` like we're doing with `onRelease` and `invalid` on line 731/732?